jdaviz/configs/cubeviz/plugins/slice/tests/test_slice.py

```diff
@@ -114,3 +114,20 @@ def test_indicator_settings(cubeviz_helper, spectrum1d_cube):
 
     sl.show_wavelength = False
     assert indicator.label.visible is False
+
+
+@pytest.mark.filterwarnings('ignore:No observer defined on WCS')
+def test_init_slice(cubeviz_helper, spectrum1d_cube):
+    cubeviz_helper.load_data(spectrum1d_cube, data_label='test')
+
+    fv = cubeviz_helper.app.get_viewer('flux-viewer')
+    sl = cubeviz_helper.plugins['Slice']
+    assert sl.slice == 1
+    assert fv.state.slices == (0, 0, 1)
+
+    # make sure adding new data doesn't revert slice to 0
+    mm = cubeviz_helper.plugins['Moment Maps']
```

Comment:
I don't get this part. You said this test is to "rather than relying on other plugins such as moment maps" but the plugin is used here.

Reply:
other _plugin tests_ (I'll update the description).  It was somewhat of a coincidence that an existing test caught this bug (and we didn't have any explicit coverage for the initial slice as far as I can tell) and took a little bit of digging to realize the issue was with the slice state and not moment maps itself.  I'm hoping this would avoid that in the future.